services/command/meeting_service.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+const request = require('request');
+const { formatDate, withoutTime } = require('./utils/date_service');
+const findTour = require('./utils/find_tour_service');
+const cron = require('./utils/create_job_service');
+const regular = require('../../regular');
+const Info = require('../../repositories/meeting-bot/info');
+const Tourist = require('../../repositories/your-tour-bot/tourist');
+const Tour = require('../../repositories/your-tour-bot/tour');
+const City = require('../../repositories/your-tour-bot/city');
+
+const end = {};
+const show = async (message, send, users, sendLocation) => {
+  const currentTour = await getTour(message, users);
+  if (currentTour) {
+    const note = await Info.getOne({ tour_id: currentTour._id });
+    if (note && note.date >= withoutTime(new Date())) {
```

Comment:
`note?.date` - optional chainig?

Reply:
![image_2021-05-27_09-19-08](https://user-images.githubusercontent.com/56652716/119775689-a2457980-becc-11eb-9052-82a8bc469856.png)

we tried, code parser is not agree